docs/layers.md

```diff
@@ -1,116 +1,510 @@
+
+* `point`: matches Point, MultiPoint
+* `line`: matches LineString, MultiLineString
+* `polygon`: matches Polygon, MultiPolygon
+
+**Tangram scene file examples:**
+
+```
+filter: { $geometry: polygon }                      # matches polygons only
+filter: { $geometry: [point, line] }                # matches points & lines
+filter: function() { return $geometry === 'line' }  # matches lines only
+```
+
+Mapnik supports geometry filtering via the special `mapnik::geometry_type` keyword.
+
+**CartoCSS Examples:**
+
+```
+#layer['mapnik::geometry_type'=1] { /* point styles */ }
+#layer['mapnik::geometry_type'=2] { /* line styles */ }
+#layer['mapnik::geometry_type'=3] { /* polygon styles */ }
+```
+
+
+#### Data updates
+
+Most Mapzen vector tile content is updated minutely from OpenStreetMap. Low and mid-zoom tiles are updated approximately monthly. Some source data rarely updates – Natural Earth updates approximately yearly.
+
+#### Changelog
+
+The current version of Mapzen vector tiles is v0.7. Our tiles are still in active development, but we try to minimize backwards incompatable breaking changes. We're also interesting in your feedback at hello@mapzen.com!
+
+If you're signed up for a Mapzen Vector Tiles API key you should receive an email notifying you of upcoming changes before they are rolled out to production.
+
+Read the full details in the project [CHANGELOG](../CHANGELOG.md).
+
+
+## Layer reference
+
+Mapzen vector tiles include 9 layers:   
+
+* `boundaries`, `buildings`, `earth`, `landuse`, `places`, `pois`, `roads`, `transit`, and `water`
+
+These individual layers are grouped into an `all` layer – use this special layer for all your general purpose mapping needs.
+
+### Boundaries & Barriers
+
+![image](images/mapzen-vector-tile-docs-boundaries.png)
+
+* Layer name: `boundaries`
+* Geometry types: `line`
+
+Combination of OpenStreetMap administrative boundaries (zoom >= 8), Natural Earth boundaries (zoom < 8), and some other `barrier` like goodies at high zooms.
+
+(below) Fence lines around the petting zoo in San Francisco are included in the `boundaries` layer.
+
+![image](images/mapzen-vector-tile-docs-barriers.png)
+
+**Gotchas:** Boundary `kind` values are not yet normalized between OpenStreetMap and Natural Earth. See Boundary kind values (line) gotchas section below for more detail.
+
+**Boundary properties (common):**
 
 * `name`
-* `kind`: one of `ocean`, `riverbank`, `dock`, `water`, `basin`, `reservoir`, `lake`, `playa`, `canal`, `dam`, `ditch`, `drain`, `river`, `stream`
-* `area`: polygon area
-* `source`: one of `naturalearthdata.com`, `openstreetmapdata.com`, `openstreetmap.org`
+* `id`
+* `kind`: mapping of OpenStreetMap's `admin_level` int values to strings like `country` and `state`, plus `aboriginal_lands` boundary type, and inclusive of some barrier and man_made tags: `city_wall` (zoom 12+), `retaining_wall`, `snow_fence` (zoom 15+), and `fence` (zoom 17+ only). Also includes raw Natural Earth values.
```

Comment:
Once the [meta-tiling at z16 changes](#478) land, we'll want to update this to say z16+ only. Maybe we'll want to highlight the `min_zoom` idea more prominently too.